funnel/templates/settings.html.jinja2

```diff
@@ -38,38 +38,38 @@
               {%- if project.current_roles.admin %}
                 {%- if transition_form %}
                   <div>
-                    <p class="mui--text-body2 zero-bottom-margin mui--text-uppercase">Status: {{ project.state.label.title }}</p>
+                    <p class="mui--text-body2 zero-bottom-margin mui--text-uppercase" data-cy="project-state">Status: {{ project.state.label.title }}</p>
                     <form action="{{ project.url_for('transition') }}" method="post" class="form-inline">
                       {{ transition_form.hidden_tag() }}
                       <p class="btn-group">
                         {% for name, transition in transition_form.transition.choices %}
-                          <button name="transition" value="{{ name }}" class="transition mui-btn mui-btn--small mui-btn--raised {% if transition.data['type'] == 'success' %} mui-btn--primary {% elif transition.data['type'] == 'danger' %}  mui-btn--danger {%- endif %}" title="{{ transition.data['title'] }}">{{ transition.data['title'] }}</button>
+                          <button name="transition" value="{{ name }}" class="transition mui-btn mui-btn--small mui-btn--raised {% if transition.data['type'] == 'success' %} mui-btn--primary {% elif transition.data['type'] == 'danger' %}  mui-btn--danger {%- endif %}" title="{{ transition.data['title'] }}"  data-cy-state="{{ name }}">{{ transition.data['title'] }}</button>
                         {% endfor %}
                       </p>
                     </form>
                   </div>
                 {%- endif %}
                 {%- if cfp_transition_form %}
                   <div>
-                    <p class="mui--text-body2 zero-bottom-margin mui--text-uppercase">{% trans %}CfP status{% endtrans %}: {{ project.cfp_state.label.title }}</p>
+                    <p class="mui--text-body2 zero-bottom-margin mui--text-uppercase" data-cy="cfp-state">{% trans %}CfP status{% endtrans %}: {{ project.cfp_state.label.title }}</p>
```

Comment:
This is incorrect use of the `trans` tag. Correct usage:

```jinja2
<p ...>{% trans status=project.cfp_state.label.title %}CfP status: {{ status }}{% endtrans %}</p>
```

Please note that the entire block is translated, not just one fragment of text.